test/source-any/05-immutability-regex-blocks-re-transfer.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+cd "$(readlink -f "$(dirname "$0")")/.."
+# shellcheck source=lib.sh
+source lib.sh
+
+step 'Test 05-immutability-regex-blocks-re-transfer'
+
+mirror <<-EOF
+  swift: { $AUTH_PARAMS }
+  jobs:
+    - from: ${SOURCE_SPEC}
+      to: { container: ${CONTAINER_BASE}-test5 }
+      only: '/$|file.txt'
+      immutable: '.*.txt'
+EOF
+
+expect test5 <<-EOF
+>> just/another/file.txt
+Hello Another World.
+EOF
```

Comment:
Where is `just/another/file.txt` coming from? I don't see it being uploaded neither in `test/lib.sh` nor anywhere above this assertion in the same file.

Upon further inspection, other tests similarly reference files in the assertions that are never uploaded, so I'm not going to repeat this note again over there.

Reply:
Yeah, I didn't change this yet. Right now all tests still need to be executed in order. When they can be executed out of order it is also very easy to parallelise them.